index-barline.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
```

Comment:
renamed the index file of the BarLine tile to index-barline.html. 

I renamed it because index.html will be used for GroupedBarChart